### DISCUSSION TITLE
Why there is black box now whenever i highlight a text using keyboard short "a"?

### DISCUSSION THREAD

**Discussion by wesmosis** [sumatrapdfreader/sumatrapdf#3739] [General]
Why there is black box now whenever i highlight a text using keyboard short "a"?

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/4676176/6a397ba3-9d55-459b-992b-f6d5d1dfe68e)

**Comment by GitHubRulesOK** [comment]
Because there were other issues and at the time that was part of the fix https://github.com/sumatrapdfreader/sumatrapdf/issues/3675 to show current ACTIVE annotation see https://github.com/sumatrapdfreader/sumatrapdf/issues/3732

**Reply by wesmosis** [reply]
Gotcha, thanks!

**Comment by meteorquake** [comment]
I came on to ask the same thing about how to switch it off. It's pretty horrendous, and for OCR'd works tends to obscure the phrase you highlight.
